dbfit-java/postgres/src/main/java/dbfit/environment/postgres/NameNormaliserPostgres.java

```diff
@@ -0,0 +1,23 @@
+package dbfit.environment.postgres;
+
+public class NameNormaliserPostgres {
+    private NameNormaliserPostgres() {
+        // utility classes should not be instanciated
+    }
+
+
```

Comment:
That empty line should should be rather moved after the closing `}` of `isEscaped`.